css/faq-style.css

```diff
@@ -95,6 +97,7 @@
 	border-bottom: 3px solid #393939;
 	font-weight: bold;
 	margin-bottom: 1em;
+	border-radius: 3px 3px 0 0;

```

Comment:
this radius is hardly noticeable. In this case, a more dramatic radius can really connect the question to the answer as a sort of unit bound as a single item. Relative em units would ensure consistency with type size. The color of the bottom border could also be changed for var(--background).
How about we match the radius to the padding on the sides, but to accentuate the effect further, we change them both to 1em:
```
padding: 1em;
border-bottom: 3px solid var(--background);
border-radius: 1em 1em 0 0;
```
font-weight and margin-bottom ok as they are.